src/game/scenes/loading.mts

```diff
@@ -143,7 +149,7 @@ export default class LoadingScene extends Phaser.Scene {
         });
         joinText.setInteractive();
         joinText.on("pointerdown", async () => {
-          if (match.matchTier.type === "tutorial") {
```

Comment:
It's a bit weird that this is how we decide whether tutorial or not tbh. I just tried adding `&tutorial=true` to a desert racer iframe for a normal match and it was going full tutorial mode, but in the end fortunately submitted 0 as the score. We can only hope that all games to this so that you can't cheat by playing a normal match in tutorial mode... 😅

Reply:
well, we always had this query parameter and that's how the fs games implemented it. if we want to consolidate this, we probably should first communicate that to fs. Should I readd the matchTier-Name to the search endpoint beforehand?

Reply:
will first wait for https://github.com/playt-net/clashparadise/pull/2867 and mend later.